app/models/service_orchestration.rb

```diff
@@ -25,8 +25,9 @@ def orchestration_stack_status
   end
 
   def deploy_orchestration_stack
+    creation_options = stack_options
```

Comment:
right, I meant that we could pass the stack_options directly to the create_stack